rainbows.rb

```diff
@@ -27,3 +31,21 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+# Let's create a rainbow of colors as an array of hashes, where each hash contains the name of the color (key) 
+# and labeled RGB values (value).
+
+
+
+new_color = mix_colors(red,green)
+puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+
+
+def print_rainbow(array)
+  array.length.times do |iteration|
+    puts array[iteration].keys.join.upcase
+  end
+end
```

Comment:
I did a refactor, taking into account this feedback.

On Wed, Mar 30, 2016 at 10:01 PM Crystal <notifications@github.com> wrote:

> In rainbows.rb
> <https://github.com/Ada-C5/rainbows/pull/5#discussion_r58001553>:
>
> > +
> > +# Let's create a rainbow of colors as an array of hashes, where each hash contains the name of the color (key)
> > +# and labeled RGB values (value).
> > +
> > +
> > +
> > +new_color = mix_colors(red,green)
> > +puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
> > +
> > +
> > +
> > +def print_rainbow(array)
> > +  array.length.times do |iteration|
> > +    puts array[iteration].keys.join.upcase
> > +  end
> > +end
>
> This is ok, but won't allow us to colorize the words later. Even if we
> don't do the colorizing, it would have been nice to write code that would
> allow adding that easily later.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/rainbows/pull/5/files/f24b31310a15349a1065123731a9c93133126bea#r58001553>
>